docs/zh_CN/guide/sinks/overview.md

```diff
@@ -22,22 +22,23 @@
 
 这些插件有预编译的二进制文件，用于主流cpu架构，如AMD或ARM。预编译的插件托管在 `https：//packages.emqx.net/kuiper-plugins/$version/$os/sinks/$type_$arch.zip` 中。例如，要获得用于 debian amd64 的 tdengine 插件，请从 `https：//packages.emqx.net/kuiper-plugins/1.4.4/debian/sinks/tdengine_amd64.zip` 安装。
 
-预定义的动作插件列表。
+预定义的动作插件列表：
 
+- [SQL](./plugin/sql.md)：写入 SQL。
 - [InfluxDB sink](./plugin/influx.md)： 写入 Influx DB `v1.x`。
 - [InfluxDBV2 sink](./plugin/influx2.md)： 写入 Influx DB `v2.x`。
 - [Tdengine sink](./plugin/tdengine.md)： 写入 Tdengine 。
-- [Image sink](./plugin/image.md)： 写入一个图像文件。仅用于处理二进制结果。
-- [Zero MQ sink](./plugin/zmq.md)：输出到 Zero MQ 。
+- [Image sink](./plugin/image.md)：写入一个图像文件。仅用于处理二进制结果。
+- [ZeroMQ sink](./plugin/zmq.md)：输出到 Zero MQ 。
```

Comment:
Does the latter Zero MQ need to be changed?

Reply:
thanks, just push another commit to update from Zmq, Zero Mq to ZeroMQ